docs/commercial/single-sign-on.rst

```diff
@@ -90,6 +89,10 @@ By default, users that Sign Up with a Google account do not have any permissions
 However, you can define which Teams users matching your company's domain email address will auto-join when they Sign Up.
 Read the following sections to learn how to grant read and admin access.
 
+You can enable this feature in your organization by going to
+"your organization detail's page" > :guilabel:`Settings` > :guilabel:`Authorization`
+and selecting :guilabel:`Google` as provider and writing your ``company.com`` domain in :guilabel:`Domain` field.
```

Comment:
```suggestion
and selecting :guilabel:`Google` as provider and specifying your Google Workspace domain in the :guilabel:`Domain` field.
```

Reply:
If we want to use "Google Workspace domain" here we should change this in the rest of the document as well.

Reply:
Also, we are using GSuite in all the other places, so that's another required change.